src/Simulations/simulation.jl

```diff
@@ -96,8 +99,18 @@ function set!(
     nothing
 end
 
+"""
+    step!(sim::AbstractSimulation, args...; kwargs...)
+
+Step forward a simulation `sim` one time step.
+"""
 step!(sim::AbstractSimulation, args...; kwargs...) =
```

Comment:
@bischtob Could I change `sim` here to `simulation` to be consistent with the `set!` function?

Reply:
Sounds good!